databases/week2/Part3.sql

```diff
@@ -0,0 +1,32 @@
+use hyf_lesson2;
+
+#All the tasks assigned to users whose email ends in @spotify.com
+SELECT task.*
+FROM user 
+JOIN user_task ON user_task.user_id=user.id
+JOIN task ON user_task.task_id=task.id
+WHERE user.email LIKE '%@spotify.com';
+
+#All the tasks for 'Donald Duck' with status 'Not started'
+SELECT task.*
```

Comment:
Here it would also be good to see the user name and the status name.

Reply:
Thanks for the comment. 👍